custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -179,7 +179,7 @@ function ServiceDeliveryDashboardController($rootScope, $scope, $http, $location
                         case "thr": return renderPercentageAndPartials(full.thr, full.thr_given_21_days, full.total_thr_candidates, 'THR');
                         case "pse": return renderPercentageAndPartials(full.pse, full.pse_attended_21_days, full.children_3_6, 'beneficiaries');
                         case "supNutrition": return renderPercentageAndPartials(full.sn, full.lunch_count_21_days, full.children_3_6, 'beneficiaries');
-                        case "cbe": return renderPercentageAndPartials(full.sn, full.num_awcs_conducted_cbe, full.num_launched_awcs, 'CBE');
```

Comment:
was the error because `full.sn` didn't exist?

Reply:
There were a couple in the end. The first one was the incorrect `case` argument, but it turns out I also misunderstood how the percent formatting worked, and I needed to do the calculation on the python side, which I added as `full.cbe`